nat-lab/run_local.py

```diff
@@ -156,6 +156,7 @@ def get_pytest_arguments(options) -> List[str]:
         if options.moose:
             marks = marks.replace("and not moose", "")
         args.extend(["-m", marks])
+    args.extend(["-k", "test_coredump"])
```

Comment:
Ignore this for now, I'll remove later once all review change requests are done and tested.